flow/layers/container_layer_unittests.cc

```diff
@@ -55,6 +63,34 @@ TEST_F(ContainerLayerTest, PaintBeforePrerollDies) {
 }
 #endif
 
+TEST_F(ContainerLayerTest, LayerWithParentHasTextureLayerNeedsResetFlag) {
+  SkPath child_path1;
+  child_path1.addRect(5.0f, 6.0f, 20.5f, 21.5f);
+  SkPath child_path2;
+  child_path2.addRect(8.0f, 2.0f, 16.5f, 14.5f);
+  SkPaint child_paint1(SkColors::kGray);
+  SkPaint child_paint2(SkColors::kGreen);
+
+  auto mock_layer1 = std::make_shared<MockLayer>(
+      child_path1, child_paint1, false /* fake_has_platform_view */, false,
+      false, true /* fake_has_texture_layer */);
+  auto mock_layer2 = std::make_shared<MockLayer>(child_path2, child_paint2);
+
+  auto root = std::make_shared<ContainerLayer>();
+  auto container_layer1 = std::make_shared<ContainerLayer>();
+  auto container_layer2 = std::make_shared<ContainerLayer>();
```

Comment:
container_layer2 is added to root, but it is never given any children. Was mock_2 supposed to be placed in container_2? Either way, the test is valid, but the existence of container_2 looks odd.

Reply:
done!